src/RestApi/StoreApi/Schemas/CartShippingRateSchema.php

```diff
@@ -202,13 +220,31 @@ public function get_item_response( $package ) {
 		// Add product names and quantities.
 		$items = array();
 		foreach ( $package['contents'] as $item_id => $values ) {
-			$items[ $item_id ] = [
+			$items[] = [
+				'key'      => $item_id,
 				'name'     => $values['data']->get_name(),
 				'quantity' => $values['quantity'],
 			];
 		}
 
+		// Generate package name.
+		$package_number       = absint( $package['package_id'] ) + 1;
```

Comment:
Why is this incremented by 1?

Reply:
Arrays have index 0 and we want to show the first as "Package 1"

Reply:
Oh I see, so `package_id` is really just an array index?

Reply:
Yup :)

Reply:
While testing with _WooCommerce Advanced Shipping Packages_, I saw it's not an array index but the ID of a WP post where the package rules are stored.

So I think we could safely remove the `+ 1` here, because they are ids instead of indexes. But if we do so, we might want to do the same in core to keep the shortcode and block working the same:

https://github.com/woocommerce/woocommerce/blob/master/includes/wc-cart-functions.php#L231

Reply:
In our case it comes from an index. There is no post for context.